index.html

```diff
@@ -1,20 +1,120 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+
+<head>
     <meta charset="utf-8">
     <meta name="description" content="Your description goes here">
     <meta name="keywords" content="one, two, three">
 
-    <title>Blank Template</title>
+    <title>Relaxr Blog</title>
 
     <!-- external CSS link -->
     <link rel="stylesheet" href="css/normalize.css">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,600,800" rel="stylesheet">
     <link rel="stylesheet" href="css/style.css">
 
-  </head>
-  <body>
-    <div class="container">
-      <!-- Replace me with your html -->
+
+</head>
+
+<body>
+    <div class="row">
+        <header>
+            <h1>Relaxr</h1>
+        </header>
+
+        <nav>
+            <ul>
+                <li>
+                    <a href="About.html">About</a>
+                </li>
+                <li>
+                    <a href="">FAQ</a>
+                </li>
```

Comment:
We didn't go over this, but for links that don't go anywhere, in order to make them valid HTML, use a '#'. So `<a href="#"></a>`. Very good markup and css tho!